### ISSUE TITLE
[StimulusBundle] ux_controller_link_tags twig function does not add the complete URI path

### ISSUE THREAD

**Issue by redflo** [symfony/ux#1104]
My base.html.twig looks like this:

```
....
        {% block stylesheets %}
            {{ ux_controller_link_tags() }}
            <link rel="stylesheet" href="{{ asset('vendor/bootstrap.min.css') }}">
            <link rel="stylesheet" href="{{ asset('styles/styles_custom.css') }}">
        {% endblock %}
....
```

Which results in:

```
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tom-select@2.2.2/dist/css/tom-select.default.css">
<link rel="stylesheet" href="/assets/@symfony/ux-dropzone/style.min-398b47e66adf4c5b513a4527c43351a4.css">
<link rel="stylesheet" href="/assets/@symfony/ux-live-component/live.min-5108f988fb2a3dbb292d6feebc9db7e8.css">
<link rel="stylesheet" href="/My-App/assets/vendor/bootstrap.min-10e855ab55e441479999bf0c59d0860b.css">
<link rel="stylesheet" href="/My-App/assets/styles/styles_custom-a11f82e8e06b4f5a63a8b16d92ce6e58.css">

```
And the ux-dropzone and ux-live-component assets cannot be delivered, The webserver gives me a 404 not found because of the missing /My-App/ in the URI. 

As discussed here:
https://symfony-devs.slack.com/archives/C01FN4EQNLX/p1694259851551709?thread_ts=1694254468.404879&cid=C01FN4EQNLX
there seems to be a missing call to the Asset component.

**Comment by weaverryan** [comment]
If https://github.com/symfony/symfony/pull/51543 is merged, I think we will be able to solve this in a different way, which will fix the issue here. In other words, this should be fixed in Symfony 6.4.

If someone wants to fix it earlier, we basically need to inject the `Packages` service into `UxControllersTwigRuntime` and wrap any calls to `$asset->publicPath` - e.g. https://github.com/symfony/ux/blob/2.x/src/StimulusBundle/src/Twig/UxControllersTwigRuntime.php#L65 - to a call like this:

```php
return $this->assetPackages->getUrl(ltrim($asset->publicPath, '/'));
```

**Comment by weaverryan** [comment]
Will be fixed in Symfony 6.4 and the first release of StimulusBundle that supports that :)
